### PR TITLE
V[0.0.1] Add Db schema desing, flyway and setup

### PR DESCRIPTION
**Pull request by geovannymcode**

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+  postgres_db:
+    container_name: "postgres_db"
+    image: "postgres"
```
Comment (sivaprasadreddy) [inline compose.yaml:4]: It is always recommended to specify the tag like "postgres:17"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
-# spring-blog
+# Spring Boot From Zero To Hero
```
Comment (sivaprasadreddy) [inline README.md:1]: This needs to be changed to "Spring Blog" and also can remove the link to YouTube video.

File: src/main/resources/db/migration/V3__create_post_table.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
+CREATE SEQUENCE post_id_seq START 1 INCREMENT BY 50;
+
+CREATE TABLE POSTS
+(
+    id                BIGINT                                                        DEFAULT nextval('post_id_seq') NOT NULL,
+    title             VARCHAR(255) UNIQUE                                  NOT NULL,
+    slug              VARCHAR(255) UNIQUE                                  NOT NULL,
+    short_description VARCHAR(255) UNIQUE                                  NOT NULL,
```
Comment (sivaprasadreddy) [inline src/main/resources/db/migration/V3__create_post_table.sql:8]: short_description need not be a unique column.

File: README.md (hunk 2 above)
Comment (geovannymcode) [reply README.md:1]: solved

File: src/main/resources/db/migration/V3__create_post_table.sql (hunk 3 above)
Comment (geovannymcode) [reply src/main/resources/db/migration/V3__create_post_table.sql:8]: solved

File: compose.yaml (hunk 1 above)
Comment (geovannymcode) [reply compose.yaml:4]: solved

**Pull request merged**